src/components/sections/projects/MyProjectVulnerabilityReport.tsx

```diff
@@ -0,0 +1,161 @@
+'use client'
+
+import Link from 'next/link'
+import {ArrowLeft, FileUp, LinkIcon, VerifiedIcon} from 'lucide-react'
+import {notFound, useRouter} from 'next/navigation'
+import {DateTime} from 'luxon'
+
+import FindingSeverityBadge from '../finding/FindingSeverityBadge'
+
+import {Button} from '@/components/ui/Button'
+import Separator from '@/components/ui/Separator'
+import {useGetFinding} from '@/lib/queries/finding/getFinding'
+import Skeleton from '@/components/ui/Skeleton'
+import {formatDate} from '@/lib/utils/common/format'
+
+type MyProjectVulnerabilityReportProps = {
+  findingId: string
+}
+
+const MyProjectVulnerabilityReport = ({
+  findingId,
+}: MyProjectVulnerabilityReportProps) => {
+  const {data, isLoading} = useGetFinding({findingId})
+  const router = useRouter()
+
+  if (isLoading) {
+    return (
+      <div className="px-24">
+        <div className="mb-12">
+          <Skeleton className="h-[72px]" />
+        </div>
+        <Separator />
+        <div className="mt-12 flex flex-col gap-3">
+          <Skeleton className="h-[72px]" />
+          <Skeleton className="h-[72px]" />
+          <Skeleton className="h-[72px]" />
+        </div>
+      </div>
+    )
+  }
+
+  if (!data) {
+    return notFound()
+  }
+
+  return (
+    <>
+      <div className="px-24">
+        <div className="flex items-center gap-12">
+          <Button
+            variant="outline"
+            size="small"
+            className="flex gap-2 uppercase"
```

Comment:
Just a note, all Buttons and also Links inside Buttons are `uppercase` by default, so no need to specify it everywhere.